Add tl;dr section for TAM and CSM roles (#18055)
PostHog/posthog.com: contents/handbook/growth/sales/csm-tam-overlay-coverage.md

## What each role focuses on

### tl;dr
 - TAM is focused on and leading commercial conversations (product adoption, renewals, expansion, etc...)

>  - CSM is focused on and leading technical conversations (efficient implementation, renewals, support escalation)

Reviewer: ```suggestion
 - CSM is focused on and leading technical conversations (efficient implementation, product training, support escalation)
```

>  - TAMs and CSMs should meet regularly (every other week / once a month) to discuss where each person is focused on the account and agree to next steps

Reviewer: ```suggestion
 - TAMs and CSMs should meet regularly (every other week / once a month) to discuss where each person is focused on the account and agree to next steps and have a shared account plan documented
```

- Cross-sell strategy and execution
- Credit discount negotiation and deal structuring for new credit purchases, invoicing
- Use case discovery, mapping products to problems
- Renewal process
- Multi-threading into new teams and stakeholders

> - Account planning (quarterly in Vitally)

Reviewer: ```suggestion
- Account planning (updated regularly in Vitally)
```
